CHANGELOG.txt

```diff
@@ -1,3 +1,8 @@
+1.0.x: 
+ - obspy.core:
+   * Trace.filter and Stream.filter don't work on masked arrays anymore because
+     which gave unpredictable results.
```

Comment:
s/because which gave/because that resulted in

or something along these lines.